examples/logger.py

```diff
@@ -133,6 +136,29 @@ def __init__(self, name: str, parent: Optional[QObject] = None):
         logger = logging.getLogger()
         logger.setLevel(logging.DEBUG)
         logger.addHandler(self.handler)
+        self.initLevelSelector(self.loggerFrame.levelSelector)
+
+    def initLevelSelector(self, box: QComboBox):
+        """Initializes a combo box for selecting level."""
+        box.addItem("DEBUG")
+        box.addItem("INFO")
+        box.addItem("WARNING")
+        box.addItem("ERROR")
+        box.addItem("CRITICAL")
+        box.activated[str].connect(self.fixLevel)
```

Comment:
Could you explain about the `activated[str]`? Is there any reference about this?
I found [`textActivated`](https://doc.qt.io/qt-5/qcombobox.html#textActivated).

Reply:
I read [activated](https://doc.qt.io/qt-5/qcombobox.html#activated).
The signal 'activated[str]' emits signal as string when comboBox's current text is changed.  
I think textActivated is more appropriate. I will use it.

Reply:
However, its argument is `int` (`index`). I wonder where you found the usage of the `[str]`.